docs/versioned_docs/version-2.0.0-beta/Enterprise/superadmin.md

```diff
@@ -0,0 +1,128 @@
+---
+id: superadmin
+title: Super Admin
+---
+
+<div className='badge badge--primary heading-badge'>Available on: Enterprise Edition</div>
+
+A Super Admin is the user who has full access to all the Workspaces, Users, and Groups of an instance. An instance can have more than one Super Admins. A Super Admin have full control over other users' workspaces and can create users, groups, and other super admins.
```

Comment:
The user details entered while setting up ToolJet will have super admin privileges